### DISCUSSION TITLE
오픈채팅방 url 체크가능?

### DISCUSSION THREAD

**Discussion by difmwng3** [storycraft/node-kakao#412] [Q&A]
메세지가 오면
해당 오픈 채팅방의 url 를 확인하고싶습니다.

함수로는 안보이는것 같던데 가능한가요 ?

**Comment by storycraft** [comment]
TalkOpenChannel 객체의 info 속성인 OpenChannelInfo객체의 openLink속성을 이용하거나 TalkOpenChannelList객체의 getOpenLink메서드에서 linkId를 이용해 OpenLink객체를 가져올 수 있습니다. 해당 객체의 linkURL속성을 이용하세요

https://www.github.com/storycraft/node-kakao/tree/stable/src%2Fopenlink%2Findex.ts
